frontend/public/components/overview/build-overview.tsx

```diff
@@ -110,7 +111,7 @@ const BuildOverviewList: React.SFC<BuildOverviewListProps> = ({ buildConfig }) =
           </div>
           {canStartBuild && (
             <div>
-              <Button bsStyle="default" bsSize="xs" onClick={onClick}>
+              <Button variant="link" isInline onClick={onClick}>
```

Comment:
This was a regular button before, not a link.

Reply:
updated